components/planner/src/dynamo/planner/utils/pre_swept_results_utils.py

```
@@ -0,0 +1,183 @@
+                updated = True
+        if updated:
+            self.save_npz_file()
+
+
+def merge_raw_data(raw_data_npz_path, configs, mode):
+    # check if mode is valid
+    # convert raw_data_file to prefill or decode npz file
+    if mode == "prefill":
+        npz = PrefillNpz(raw_data_npz_path, configs)
+    elif mode == "decode":
+        npz = DecodeNpz(raw_data_npz_path, configs)
```

Ruff (0.13.1)

Local variable `npz` is assigned to but never used

Remove assignment to unused variable `npz`

(F841)